src/mumble/ManualPlugin.cpp

```diff
@@ -43,6 +43,7 @@ Manual::Manual(QWidget *p) : QDialog(p) {
 
 	qgvPosition->viewport()->installEventFilter(this);
 	qgvPosition->scale(1.0f, 1.0f);
+	qgvPosition->setRenderHints(QPainter::Antialiasing);
```

Comment:
I would set this in `ManualPlugin.ui`, right now it sets `QPainter::TextAntialiasing`.

Reply:
https://github.com/mumble-voip/mumble/blob/20cfde7860c69dcd12ad544d301210bd4965bd52/src/mumble/ManualPlugin.ui#L31-L69

The setting is not present because we didn't alter the default value.

You have to add:
```
<property name="renderHints">
 <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
</property>
```

Or, if you have Qt Creator (or Qt Designer):
![](https://user-images.githubusercontent.com/5897523/71326545-9a75ee80-24fc-11ea-8a64-1e3eaef3ba2d.png)

Reply:
Done